### PR TITLE
Scaling Lightning Blog post

### PR DESCRIPTION
**Pull request by bonomat**
We can add a follow-up blog post to this one on Ark and Fedi which both address this issue as well. 

Link to draft: https://deploy-preview-42--elaborate-tiramisu-8c63d3.netlify.app/blog/scale-btc-to-billions

### CODE REVIEW

File: blog/2023-06-06-scaling-bitcoin-to-billions.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+When you use a custodial Lightning wallet, you are able to interact with Lightning rails and other payment channels on the network, but you’re not holding the funds yourself.
+
+You are merely sending a request to a third-party provider that routes the payments on your behalf and showcase the balance on your screen. You have no way of knowing if that balance exists or if it’s backed by bitcoin until you move it into a non-custodial wallet or swap it for an on-chain UTXO via a [submarine swap](https://docs.lightning.engineering/the-lightning-network/multihop-payments/understanding-submarine-swaps).
+
+A custodial wallet is no different from a banking app you have on your phone right now, except instead of bank crediting ledgers, the provider behind the wallet updates Lightning balances on your behalf.
+There is, therefore, a lot of trust between you and the wallet provider and many ways in which this trust can be abused.
+
+> At 10101, we do not endorse custodial (Lightning) wallets as by the definition, there is no such thing as a custodial bitcoin. Your "bitcoin" on an exchange or in a custodial wallets are IOUs and not real bitcoin.
```
Comment (holzeis) [inline blog/2023-06-06-scaling-bitcoin-to-billions.md:45]:
```suggestion
> At 10101, we do not endorse custodial (Lightning) wallets as by the definition, there is no such thing as a custodial bitcoin. Your "bitcoin" on an exchange or in a custodial wallets are IOUs and not real Bitcoin.
```

File: blog/2023-06-06-scaling-bitcoin-to-billions.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Additionally, running a Lightning Node needs
+
+- high capital upfront; and
+- technical expertise
+
+Requirements that will be unrealistic for some, especially those who only wish to receive funds with low friction.
+
+## Why not custodial Lightning channels?
+
+The simplest way to interact with the Lightning Network is to use a custodial wallet; this could be through an app like the [Wallet of Satoshi](https://www.walletofsatoshi.com/) or an exchange account you might have with a provider like CashApp.
```
Comment (holzeis) [inline blog/2023-06-06-scaling-bitcoin-to-billions.md:37]: Is it the simplest way though? I'd argue that Phoenix and Breez are also very simple to ineract with the Lightning Network. What we need though are LSPs and inbound liquidity, that makes it complicated.

File: blog/2023-06-06-scaling-bitcoin-to-billions.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- One side of such a channel is called a “Host” because it’s the side where the actual money is stored.
+- The other side is called a "Client", and it’s the one who trusts the Host.
+
+Established Hosted channels can be announced to the Lightning network where the funds in the hosted channel are a pure Host’s IOU to the client which is not automatically enforceable on-chain but is still auditable.
+
+Host’s balance obligations are cryptographically provable at all times, so the only type of scam a Host can viably perform is an exit scam.
+So instead of trying a centralised third party that will consolidate their internal database with the balances in their wallet, you have more assurance that real bitcoin is backing these balances and transactions without needing to redeem it to prove the funds are real.
+
+However, there is still trust invovled between the "Client" and the "Host".
+
```
Comment (holzeis) [inline blog/2023-06-06-scaling-bitcoin-to-billions.md:62]: We should at least mention the idea of Greenlight.

**Review by holzeis — APPROVED** [review]
I like it!

File: blog/2023-06-06-scaling-bitcoin-to-billions.md (hunk 3 above)
Comment (bonomat) [reply blog/2023-06-06-scaling-bitcoin-to-billions.md:62]: Greenlight is not hosted channels as described here. Greenlight is a split between running the node (running on a server) and keeping the private keys on a different device (phone)

**Comment by bonomat** [comment]
waiting until tomorrow to publish it... looks like one can't schedule blog posts after all...

**Pull request merged**